packages/wrangler/e2e/dev.test.ts

```diff
@@ -285,10 +286,15 @@ describe("basic dev python tests", () => {
 					compatibility_date = "2023-01-01"
 					compatibility_flags = ["python_workers"]
 			`,
+			"arith.py": dedent`
```

Comment:
can we pls use the full noun here? `arith` might not be immediately meaningful for everyone